Dockerfile.bootstrap-prefix-centos8

```diff
@@ -1,8 +1,8 @@
-FROM centos:8.2.2004
+FROM centos:8
```

Comment:
@bedroge Hmm, I'm favor of keeping this fixed to a specific version.

Also, how about switching to Rocky Linux, since CentOS 8.5 is basically the end of road (and CentOS Stream doesn't seem like a good idea to me)

https://hub.docker.com/r/rockylinux/rockylinux/tags

Reply:
Ah, that would rule out `ppc64le` though...
Nevermind then, and I guess we can live with `8` too, which version we pull in just depends on when the container image gets built.